perma_web/perma/models.py

```diff
@@ -1109,6 +1109,64 @@ def can_view_usage_plan(self):
         return not self.nonpaying or (self.is_registrar_user() and not self.registrar.nonpaying)
 
 
+    ### merging accounts ###
+
+    def copy_memberships_from_users(self, users):
+        original_orgs = set(self.organizations.all())
+
+        orgs = set()
+        registrars = set()
+        for user in users:
+            if user.registrar_id:
+                registrars.add(user.registrar_id)
+            else:
+                orgs.update(user.organizations.all())
+
+        if orgs or registrars:
+            assert not (orgs and registrars), f"This set of users includes both org and registrar users: {', '.join([user.id for user in users])}."
+            if registrars:
+                assert len(registrars) == 1
```

Comment:
I'm using `assert`s throughout; I should catch and record in on of the higher level functions, instead of just borking.